packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -2,11 +2,14 @@
 
 ## Unreleased
 
+* Added
+  * Added aria-label custom prop and ariaDescribedBy first class prop for `terra-collapsible-menu-view` and `terra-menu` to help describe the intent of the group information.
```

Comment:
```suggestion
  * Added `aria-label`  and `ariaDescribedBy` for `terra-collapsible-menu-view` and `terra-menu` to help describe the intent of the group information.
```

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1841/commits/4cf44699ba0fff5480d30bd3f25b1b708a7b6fa6

Reply:
Update it to

>  Added `aria-label`  and `ariaDescribedBy` props

and it looks good!

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1841/commits/3d4827715d52d488e740adba63a7c12fd28bb4a8